server/proxy/sockproxy/doc.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package sockproxy is an proxy at the socket level used to proxy http and grpc requests,
```

Comment:
Why change name from multiplexer to proxy?

Reply:
yeah, it is more a multiplexer than a proxy